db/migrate/20160416105605_add_last_active_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastActiveAtToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :last_active_at, :timestamp
```

Comment:
I don't think this should ever be null? At worst case they were last active when the signed in for the first time.